src/main/scala/org/zalando/spearheads/innkeeper/services/RoutesService.scala

```diff
@@ -18,9 +18,15 @@ import scala.concurrent.{ Future, ExecutionContext }
 class RoutesService @Inject() (implicit val executionContext: ExecutionContext,
                                val routesRepo: RoutesRepo) {
 
-  def createRoute(route: NewRoute, createdAt: LocalDateTime = LocalDateTime.now()): Future[Option[Route]] = {
+  def createRoute(route: RouteIn, createdAt: LocalDateTime = LocalDateTime.now()): Future[Option[RouteOut]] = {
 
-    val routeRow = RouteRow(None, route.toJson.prettyPrint, createdAt)
+    val routeRow = RouteRow(id = None,
+      name = route.name.name,
+      routeJson = route.route.toJson.prettyPrint,
+      createdAt = createdAt,
+      description = route.description,
+      activateAt = route.activateAt.getOrElse(createdAt.plusMinutes(5))
```

Comment:
all skipper instances should have time to initialize their data structures before the routes get activated...